doc/usage/unitpackage_usage.md

```diff
@@ -213,6 +275,22 @@ A plot with rescaled axis is obtained by rescaling the entry first.
 entry.rescale({'E':'mV', 'j':'uA / cm2'}).plot(x_label='t', y_label='j')
 ```
 
+We can also use the matplotlib interface.
+
+```{code-cell} ipython3
+import matplotlib.pyplot as plt
+
+fig, ax = plt.subplots(1,1)
+
+x = 'E'
+y = 'j'
+
+entry.df.plot(x, y, ax=ax)
+plt.title(entry.identifier)
+ax.set_xlabel(f"{x} [{entry.field_unit(x)}]")
+ax.set_ylabel(f"{y} [{entry.field_unit(y)}]")
+```
+
```

Comment:
Not related to this PR, but I think this is the more common and familiar interface that people use for plotting.